molgenis/molgenis-emx2
# feat: cohorts, data sources and variables from nested networks are also included in current network views

closes https://github.com/molgenis/GCC/issues/521

What are the main changes you did:
- added partOfNetworks refback to get what networks a network is part of
- used that to in graphql queries to include cohorts, variables, data sources in counts and lists

how to test:
- use the networkOfnetworks and compare results before and after; you should see +1 cohorts, variables to be included.

todo:
- [x] add e2e test for these specific changes
- [x] variable counts are correct but link to variable harmonisation details is broken in variables in nested network
